lib/modules/manager/mise/backends.ts

```diff
@@ -0,0 +1,223 @@
+import is from '@sindresorhus/is';
+import { regEx } from '../../../util/regex';
+import { CrateDatasource } from '../../datasource/crate';
+import { GitRefsDatasource } from '../../datasource/git-refs';
+import { GitTagsDatasource } from '../../datasource/git-tags';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import { GoDatasource } from '../../datasource/go';
+import { NpmDatasource } from '../../datasource/npm';
+import { NugetDatasource } from '../../datasource/nuget';
+import { PypiDatasource } from '../../datasource/pypi';
+import { normalizePythonDepName } from '../../datasource/pypi/common';
+import { RubygemsDatasource } from '../../datasource/rubygems';
+import type { PackageDependency } from '../types';
+import type { MiseToolOptionsSchema } from './schema';
+
+export type BackendToolingConfig = Omit<PackageDependency, 'depName'> &
+  Required<
+    | Pick<PackageDependency, 'packageName' | 'datasource'>
+    | Pick<PackageDependency, 'packageName' | 'skipReason'>
+  >;
+
+/**
+ * Create a tooling config for aqua backend
+ * @link https://mise.jdx.dev/dev-tools/backends/aqua.html
+ */
+export function createAquaToolConfig(
+  name: string,
+  version: string,
+): BackendToolingConfig {
+  // mise supports http aqua package type but we cannot determine it from the tool name
+  // An error will be thrown afterwards if the package type is http
+  // ref: https://github.com/jdx/mise/blob/d1b9749d8f3e13ef705c1ea471d96c5935b79136/src/aqua/aqua_registry.rs#L39-L45
+  return {
+    packageName: name,
+    datasource: GithubTagsDatasource.id,
+    // Trim the leading 'v' from both the current and extracted version
+    currentValue: version.replace(/^v/, ''),
```

Comment:
```suggestion
    currentValue: version.replace(regEx(/^v/), ''),
```
always use our regex helper